### PR TITLE
Fixes non-clothing mask slot items runtiming breath, also makes gas filtering a defined bitfield like it should be

### LINKED ISSUE #60837: non mask items that go on mask slot runtime and break breathing
## Round ID:

 Reproduced locally

## Reproduction:

 
Give self oxyloss by any means
equip a rose or harmonica on mask slot
game will start runtiming and you won't regain the oxyloss damage you took.
the runtimes point to some code about mask filters in lungs.dm so maybe it was broken by #58529?

### PR DESCRIPTION
**Pull request by MrMelbert**
## About The Pull Request

- Fixes #60837 . Lung breath checked only if the user HAD a mask slot item, not if they had a CLOTHING mask slot item. Which runtimed if a non-clothing mask item was equipped.
- Added a few parenthesis for clarity.
- Puts `GAS_FILTERING` clothing flag in the clothing flag bitfield so it actually shows up in VV.  It did not show up while testing this fix which confused me, so now it does.

## Why It's Good For The Game

Bug fix, roses no longer prevent you from breathing 

## Changelog
:cl: Melbert
fix: Some mask slot items won't prevent you from breathing properly due to a runtime error
/:cl:

### CODE REVIEW

**Comment by Mothblocks** [comment]
it is fairly stupid `wear_mask` being wrong is just normal behavior

File: code/_globalvars/bitfields.dm [hunk 1]
```diff
@@ -90,6 +90,7 @@ DEFINE_BITFIELD(clothing_flags, list(
 	"THICKMATERIAL" = THICKMATERIAL,
 	"VOICEBOX_DISABLED" = VOICEBOX_DISABLED,
 	"VOICEBOX_TOGGLABLE" = VOICEBOX_TOGGLABLE,
+	"GAS_FILTERING" = GAS_FILTERING,
```
Comment (Mothblocks) [inline code/_globalvars/bitfields.dm:93]: Move this into ABC order like the rest of them, it looks pretty.

**Review by Mothblocks — APPROVED** [review]

**Comment by maxymax13** [comment]
gameplay change, spanish people no longer die. please wait until september

**Comment by Mothblocks** [comment]
donda esta la biblioteca

**Pull request merged**